presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveMarkDistinctQueries.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import org.testng.annotations.Test;
+
+public class TestHiveMarkDistinctQueries
+        extends AbstractTestHiveQueries
+{
+    private static final String DISTINCT_AGGREGATE = "WITH sample AS ( SELECT * FROM (VALUES (1,1,1), (1,1,2), (1,1,1), (2,1,2), (2,2,1), (2,3,2)) AS account (c1, c2, c3) ) SELECT c1,sum(distinct c2),sum(distinct c3) from sample group by c1";
```

Comment:
Why is this private static final variable and can't be local to testDistinctAggregate ?

Do you anticipate many tests for MarkDistinct Operator ? Do we need a new file for this ?